### PR TITLE
Adds servo and relay.

### PR DESCRIPTION
**Pull request by tcr**

### CODE REVIEW

File: servo-pca9685/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+use tessel::Tessel;
+use std::thread::sleep;
+use std::time::Duration;
+
+fn main() {
+    // Acquire port A.
+    let (port_a, _) = Tessel::ports().unwrap();
+
+    // Create the accelerometer object and connect to the sensor.
+    let mut servos = ServoArray::new(port_a, false, false);
```
Comment (rwaldron) [inline servo-pca9685/src/main.rs:17]: Without looking at the source, I would have no idea what `false, false` meant :\

File: servo-pca9685/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+use servo_pca9685::ServoArray;
+use tessel::Tessel;
+use std::thread::sleep;
+use std::time::Duration;
+
+fn main() {
+    // Acquire port A.
+    let (port_a, _) = Tessel::ports().unwrap();
+
+    // Create the accelerometer object and connect to the sensor.
```
Comment (rwaldron) [inline servo-pca9685/src/main.rs:16]: Typo

(sorry, I keep hitting the green button)

File: servo-pca9685/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+const MAX: u16 = 4096;
+//const MODE1 = 0x0;
+//const PRE_SCALE = 0xFE;
+
+const I2C_ID: u8 = 0x73;
+
+#[allow(dead_code)]
+pub struct ServoArray<'a> {
```
Comment (rwaldron) [inline servo-pca9685/src/lib.rs:29]: Is this actually supposed to be a `ServoArray`? or Just `Servo`?

File: servo-pca9685/src/main.rs (hunk 1 above)
Comment (tcr) [reply servo-pca9685/src/main.rs:17]: Agreed, opening an issue

File: servo-pca9685/src/lib.rs (hunk 3 above)
Comment (tcr) [reply servo-pca9685/src/lib.rs:29]: Actually Servo array (a physical array of servos) but I don't like the name. Opening an issue

**Pull request merged**